### ISSUE TITLE
[Enhancement] Do not follow align indicator

### ISSUE THREAD

**Issue by emisjerry** [pistacchio/obsidian-enhanced-tables#3]
It seems like that table don't follow the `:---`, `:---:' and `---:' align symbol in Reading mode.

```
| Id  | Number column | Date       | Rating | Color  | Formatted                        | Hidden             |   真假值    |
| --- | ------------: | ---------- | :----: | ------ | -------------------------------- | ------------------ | :------: |
| 1   |           500 | 2024-03-05 |   2    | red    | _**bold**_                       | Text you won't see |          |
| 2   |          1000 | 2024-04-10 |   5    | orange |                                  |                    |    no    |
| 3   |          1501 | 2024-03-15 |   1    | yellow | <font color="#ff0000">red</font> |                    |   yes    |
| 4   |         10000 | 2024-05-01 |   4    | green  | ~~strike~~                       |                    | whatever |
| 5   |         12000 | 2024-05-02 |   3    | blue   | ~~strike~~                       |                    |   yes    |
| 6   |         13000 | 2024-05-03 |   4    | purple | `{js} alert('123');`             |                    |    no    |
```

![image](https://github.com/pistacchio/obsidian-enhanced-tables/assets/131511/dd688e18-416f-4a4b-b43d-e3386360b61e)

**Comment by emisjerry** [comment]
I've found a solution by assigning style:

```css
   td:nth-child(2) {
     text-align: right !important;
     color: red;
   }
```

**Comment by pistacchio** [comment]
Fixed in [1.0.4](https://github.com/pistacchio/obsidian-enhanced-tables/releases/tag/1.0.4)
